docs/_layouts/guide.njk

```diff
@@ -2,21 +2,19 @@
 layout: default.njk
 ---
 {% include "page-title.njk" %}
-<div class="p-lg-16">
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-md-6">
+      {% include "page-contents.njk" %}
+    </div>
+  </div>
+</div>
+<div class="content-container bg-less-bright">
```

Comment:
Note that this new background color will not show until the "Extends" branch update is in place.